R-package/R/saveRDS.lgb.Booster.R

```diff
@@ -77,6 +79,8 @@ saveRDS.lgb.Booster <- function(object,
       , refhook = refhook
     )
 
+    return(invisible(NULL))
```

Comment:
instead of doing this, can you please just add a single `return(invisible(NULL))` outside of the if-else  block?